scanner/src/funTest/kotlin/scanners/FileCounterTest.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.scanner.scanners
+
+import com.here.ort.scanner.Main
+import com.here.ort.utils.safeDeleteRecursively
+import com.here.ort.utils.searchUpwardsForSubdirectory
+
+import io.kotlintest.Description
+import io.kotlintest.TestResult
+import io.kotlintest.matchers.shouldBe
+import io.kotlintest.specs.StringSpec
+
+import java.io.File
+
+class FileCounterTest : StringSpec() {
+    private val rootDir = File(".").searchUpwardsForSubdirectory(".git")!!
```

Comment:
This work-around should be removed now, also see https://github.com/heremaps/oss-review-toolkit/pull/566.